channels/fr.m3u

```diff
@@ -103,8 +101,6 @@ https://artelive-lh.akamaihd.net/i/artelive_de@393591/master.m3u8?set-segment-du
 http://artelive-lh.akamaihd.net/i/artelive_fr@344805/master.m3u8?set-segment-duration=quality
 #EXTINF:-1 tvg-id="Atelierdeschefs.fr" tvg-name="Atelier des chefs" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/laArg26.png" group-title="",Atelier des chefs (720p)
 https://alchimie-atelier-des-chefs-1-fr.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="BeurTV.fr" tvg-name="Beur TV" tvg-country="DZ" tvg-language="" tvg-logo="https://i.imgur.com/KQcT9cz.png" group-title="",Beur TV
-https://cdn02.hta.dz/abr_htatv/Beur_TV/htatv/Beur_TV_720p/chunks.m3u8
```

Comment:
Like the other Algerian streams, not 24/7.

Reply:
Currently streaming